docs/separate-benchmark-module.md

```diff
@@ -0,0 +1,102 @@
+# Creating Separate Module for Benchmarks
+
+This guide will walk you through the process of creating a separate module for benchmarks in your Kotlin project. 
+Creating a separate module for benchmarks is especially beneficial when you are integrating benchmarks into an existing project. 
+Here are a few advantages of doing so:
+
+1. **Organization**: It helps maintain a clean and organized project structure. Segregating benchmarks from the main code makes it easier to navigate and locate specific code segments.
+
+2. **Isolation**: Separating benchmarks ensures that the benchmarking code does not interfere with your main code or test code. This isolation guarantees accurate measurements without unintentional side effects.
+
+3. **Flexibility**: Creating a separate module allows you to manage your benchmarking code independently. You can compile, test, and run benchmarks without impacting your main source code.
+
+## Step-by-step Setup Guide
+
+Below are the step-by-step instructions to set up a separate module for benchmarks in both Kotlin Multiplatform and Kotlin/JVM projects:
+
+### Kotlin Multiplatform Project
+
+Follow these steps to set up a separate compilation for benchmarks:
+
+1. **Define New Compilation**:
```

Comment:
Here, "Define New Compilation" is like a section heading, so I'm not sure if it should ends with a colon.

Reply:
I don't have strong opinion in this regard. But it does look better without colon.
Colons used in section headings are removed now.